WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -159,18 +157,21 @@ fortytwo <- 42
 fortytwo
 ```
```

Comment:
Somewhere here you have this line:
This is one of the many reasons why you should always work with a script and annotate it: it is your life line in case you make a mistake, lose objects, modify data elements, etc.

That's actually an important point that we should stress somewhere, but not here. You can leave this here. But we should not forget to mention that using a script is crucial. Probably when we explain all the windows in RStudio and show the difference between console and script!